MonoidalCategories/init.g

```diff
@@ -4,4 +4,32 @@
 # Reading the declaration part of the package.
 #
 
+## Monoidal
+ReadPackage( "MonoidalCategories", "gap/MonoidalCategoriesDoc.gd" );
+ReadPackage( "MonoidalCategories", "gap/MonoidalCategoriesTensorProductAndUnit.gd" );
 ReadPackage( "MonoidalCategories", "gap/MonoidalCategories.gd" );
+
+ReadPackage( "MonoidalCategories", "gap/DistributiveMonoidalCategories.gd" );
+
+ReadPackage( "MonoidalCategories", "gap/BraidedMonoidalCategoriesDoc.gd" );
+ReadPackage( "MonoidalCategories", "gap/BraidedMonoidalCategories.gd" );
+
+ReadPackage( "MonoidalCategories", "gap/SymmetricMonoidalCategoriesDoc.gd" );
+
+## Closed Monoidal
+ReadPackage( "MonoidalCategories", "gap/ClosedMonoidalCategoriesDoc.gd" );
+ReadPackage( "MonoidalCategories", "gap/ClosedMonoidalCategories.gd" );
+
+ReadPackage( "MonoidalCategories", "gap/SymmetricClosedMonoidalCategoriesDoc.gd" );
+
+ReadPackage( "MonoidalCategories", "gap/RigidSymmetricClosedMonoidalCategoriesDoc.gd" );
+ReadPackage( "MonoidalCategories", "gap/RigidSymmetricClosedMonoidalCategories.gd" );
+
+## Coclosed Monoidal
```

Comment:
Also, please delete these traces of coclosed monoidal categories here (note that you read files that do not yet exist).

Reply:
OK.